jmclient/jmclient/jsonrpc.py

```diff
@@ -134,10 +136,10 @@ def queryHTTP(self, obj):
                                              repr(exc))
             break
 
-    def call(self, method, params):
+    def call(self, method: str, params: dict):
```

Comment:
Shouldn't `params` be of type list? I see we call it like `self.jsonRpc.call('foo', [bar])`. We can also put `Any` if we are not sure about the return type.

Reply:
Agree with `list`. That is the assumption in `BitcoinCoreInterface._rpc()`; if it's wrong anywhere we should fix it.

Reply:
Agree with `list`. That is the assumption in `BitcoinCoreInterface._rpc()`; if it's wrong anywhere we should fix it.

Reply:
According to JSON-RPC specs, it should be `Union[dict, list]`. And Bitcoin Core also accepts both positional and named parameters.

Reply:
Addressed in 15c099a51e54f6e8666e07affceb208942ceb2ba

Reply:
> According to JSON-RPC specs, it should be Union[dict, list]. And Bitcoin Core also accepts both positional and named parameters.

OK makes sense, and it doesn't conflict with our usage so :+1: